api-server/src/main/resources/swagger.json

```diff
@@ -733,6 +733,48 @@
                     }
                 }
             },
+            "patch": {
+                "tags": [
+                    "enmasse_v1beta1",
+                    "addresses"
+                ],
+                "description": "patches (RFC6902) the specified Address",
+                "operationId": "patchesEnmasseV1alpha1NamespacedAddress",
+                "consumes": [
+                    "application/json-patch+json"
```

Comment:
This version of Swagger doesn't permit the definition of requests that support many payload types.  I made an arbitrary choice of RFC6092  I read that OpenAPI v3 brings this ability.